# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../arora/prefix.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- arora-0.11.0/install.pri	2010-09-27 00:12:17.000000000 -0230
+++ arora-0.11.0/install.pri	2010-10-28 14:11:54.000000000 -0230
@@ -1,7 +1,7 @@
 
 unix {
     isEmpty(PREFIX) {
-        PREFIX = /usr
+        PREFIX = /opt
     }
     BINDIR = $$PREFIX/bin
 
